lawnchair/src/app/lawnchair/ui/preferences/destinations/HomeScreenPreferences.kt

```diff
@@ -85,6 +87,15 @@ fun HomeScreenPreferences(
                     enabled = lockHomeScreenAdapter.state.value.not(),
                 )
             }
+            ClickablePreference(
+                label = stringResource(id = R.string.remove_all_views_from_home_screen),
+                confirmationText = stringResource(id = R.string.remove_all_views_from_home_screen_desc),
+                onClick = {
+                    scope.launch {
+                        LauncherAppState.getInstance(context).clearAllViewsFromHomeScreen()
+                    }
+                },
+            )
```

Comment:
I'd suggest moving this below the "Layout" section, inside it's own container (similar to the "Reset custom icons" preference).

```
PreferenceGroup {
    ClickablePreference(...)
}
```